src/components/TakeAction.vue

```diff
@@ -78,17 +78,16 @@
         </v-expansion-panel-header>
 
         <v-expansion-panel-content>
-          <sign-name />
+          <sign-name @form-filled="handleFormFilled" />
         </v-expansion-panel-content>
-
         <v-expansion-panel-content>
           <v-btn
             width="160"
-            dark
             color="primary"
+            :disabled="!formFilled"
             @click="nextPage({ userData })"
           >
-            Next
+            {{ formFilled ? 'Next' : 'Next' }}
```

Comment:
If the button text is always `Next`, is the ternary operator really necessary? 😃

Reply:
lol 😃